commands/client-list.md

```diff
@@ -34,6 +34,11 @@ Here is the meaning of the fields:
 * `user`: the authenticated username of the client
 * `redir`: client id of current client tracking redirection
 * `resp`: client RESP protocol version.
+* `lib-name`: The client library name as set by `CLIENT SETINFO`
+* `lib-version`: The client library version as set by `CLIENT SETINFO`
+* `tot-net-in`: Total network input bytes read from this client. Added in Valkey 8.0
```

Comment:
So `'lib-name` and `lib-version` work today, but the `tot-*` won't be added until your PR is merged and 8.0 is released? I feel like we should seperate this into two PRs. One we can merge today and one that depends on valkey-io/valkey#327 and 8.0 release.

Reply:
I agree.

Reply:
There used to be a label for this. `waiting-for-upstream`